### DISCUSSION TITLE
Not getting minor update

### DISCUSSION THREAD

**Discussion by Justineo** [renovatebot/renovate#13093] [Request Help] [answered]
Hi! Thank you for this amazing tool!

I'm using the Renovate GitHub Action and configured my project to receive updates from a specific package ([config file](https://github.com/ecomfe/veui/blob/7381bac6e1ae6910e9a8375bb10e8d96fc491a60/.github/renovate.json)). It correctly made a PR for the major version but after the target package releases a minor version, Renovate doesn't make PRs for that.

I'm wondering if there some misconfiguration for my project. Any help would be appreciated.

**Comment by rarkins** [comment, marked as answer]
You should probably look at "rangeStrategy" in the docs

**Reply by Justineo** [reply]
Thank you! I believe this is what I'm looking for.
